src/features/wallet/scan/ScanView.tsx

```diff
@@ -154,9 +154,7 @@ const ScanView = ({ scanType = 'payment', showBottomSheet = true }: Props) => {
                   {t('send.scan.send')}
                 </Text>
                 <Text marginBottom="xs">{t('send.scan.send_description')}</Text>
-                <Text variant="body2Bold" color="blueMain">
-                  {t('send.scan.learn_more')}
-                </Text>
+                <Text variant="body2Bold" color="blueMain" />
```

Comment:
We can actually just remove these Text elements entirely if they are no longer being used.